pylabber/urls.py

```diff
@@ -40,6 +40,4 @@
 ]
 
 if settings.DEBUG:
-    urlpatterns += static(
-        settings.MEDIA_URL, document_root=settings.MEDIA_ROOT
-    )
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Please make sure your VSC preferences include:
```javascript
  "python.formatting.provider": "black",
  //// Default black line length is 88, this sets it to 79 in order to comply
  //// with flake8.
  "python.formatting.blackArgs": ["--line-length=79"],
```

Reply:
It is, maybe in the wrong place

Reply:
It can be either in the user settings under `~/.config/Code/User/settings.json` or in the workspace's settings under `<project_dir>/.vscode/settings.json`.

Reply:
Done.